ui/app/AppLayouts/Wallet/addaccount/panels/WatchOnlyAddressSection.qml

```diff
@@ -0,0 +1,65 @@
+import QtQuick 2.14
+import QtQuick.Controls 2.14
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Controls.Validators 0.1
+
+import utils 1.0
+
+import "../stores"
+
+Column {
+    id: root
+
+    property AddAccountStore store
+
+    function reset() {
+        addressInput.reset()
+    }
+
+    StatusInput {
+        id: addressInput
+        width: parent.width
+        maximumHeight: Constants.addAccountPopup.itemHeight
+        minimumHeight: Constants.addAccountPopup.itemHeight
+        label: qsTr("Ethereum address or ENS name")
+        placeholderText: "e.g.0x95222293DD7278Aa3Cdd389Cc1D1d165CCBAfe5"
```

Comment:
@stefandunca that's just a placeholder text representing some random address. Or you mean to translate it because of "e.g." (if so, we can translate or delete it).

Reply:
I'd remove the "e.g." part

Reply:
Because of the `e.g.`. @caybro's recommendation seems a better fit though.

Reply:
@stefandunca it's like that by design will check with Ben.